README.md

```diff
@@ -31,21 +31,23 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+*  Document Object Model. A structure built from the HTML, resembling OO-style objects, with an API available to JS running in the browser.
 2. What is an event?
-
+*  An object created by an element in the DOM, based on any of a huge variety of (mostly user-driven) conditions.
 3. What is an event listener?
-
+*  A bit of code associated with a given element that invokes a callback when the specified type of event occurs.
 4. Why would we convert a NodeList into an Array?
-
+*  Access to the methods associated with the Array prototype.  NodeList only has access to forEach.  And the Array prototype methods are amazing.
 5. What is a component?
+*  A way of defining/setting up behaviors for a specific type of element on your page in a reusable way, by passing them into a larger object with attatched methods.
```

Comment:
This is how I answered this question: ```A component is the intertwined html/css/js that is used to make objects on the window smarter. A component is said to be the intertwined use of html/css/js because, the js code that makes the window smarter refers to both the html and css of the window to make an object do what you want it to do (e.g. adding/removing a class name to a class in js in order to add/remove a specific style in the css.)```